cmd/diff.go

```diff
@@ -35,10 +35,10 @@ var diffCmd = &cobra.Command{
 	Long:  `Compares two images using the specifed analyzers as indicated via flags (see documentation for available ones).`,
```

Comment:
We should probably document the available image name flags somewhere, does it make sense here?

Reply:
yeah I should put it here. should also go in the README somewhere